examples/ingress-controllers/nginx/data.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_iam_policy_document" "this" {
+  statement {
+    sid       = "ListBucket"
+    effect    = "Allow"
+    resources = ["arn:aws:s3:::example-log-bucket-name"]
```

Comment:
Questions might raise that this bucket doesn't exist. Either we have to create a random bucket and use it in the example so that it won't use the same name for all the users. Also, we need to add annotations to the `values.yaml` to enable NLB logging and point to S3 bucket. Our current defaults only creates NLB

Or For now, we can remove this policy since we are not using this policy in the example. We can update this pattern to leverage NLB logs for S3 and ACM using external DNS later

Reply:
I'll remove it for now to keep it simple.